JetUtilities/src/JRAEvent.cc

```diff
@@ -84,8 +84,8 @@ void JRAEvent::MakeTree(TTree* tree)
    fChain->Branch("evt", &evt, "evt/L");
    fChain->Branch("nref", &nref, "nref/b");
    fChain->Branch("refrank", "vector<UChar_t>", &refrank);
+   fChain->Branch("refpdgid", "vector<Int_t>", &refpdgid);
```

Comment:
This change means that the `refpdgid` branch will always be loaded. Is that want you want?